assets/personal reference/TODO.vhou

```diff
@@ -0,0 +1,21 @@
+--Update all locales: Done until next coli update!
+
+--New coloring system: Colors relate to element rather than enemy coloration (easier to do)
+LIGHT: #d3db54
+SHADOW: #8026b1
+ICE: #a3d5d0
+FIRE: #d38200
+WATER: #162fa6
+ARCANE: #ff00d6
+EARTH: #886949
+PLAGUE: #bf0c0c
+NATURE: #008f02
+LIGHTNING: #00c3b7
+WIND: #6cb500
+NEUTRAL/NOTN: #484848
```

Comment:
Love love love the changeover to elemental colors. That said, and I'm sure the older version had the problem too, there's some readability issues because of the white text:

![image](https://user-images.githubusercontent.com/19622123/90526777-a6ff2500-e125-11ea-9267-db856b06ddc8.png)

Easiest solution would probably be to darken/saturate the lighter colors a bit. Again, probably no worse than the old version, but now that the colors are programmatically assigned, might be worth playing with :)

Reply:
Adding a commit to address this + the gradient background looking weird